rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -2987,6 +3020,87 @@ extern "C" rmw_ret_t rmw_get_node_names(
   return RMW_RET_BAD_ALLOC;
 }
 
+extern "C" rmw_ret_t rmw_get_node_names_with_security_contexts(
```

Comment:
@ivanpauno should this (and every other change in this patch) be conditionally compiled based on RMW version too?

Reply:
Thanks for the reminder, see https://github.com/ros2/rmw_cyclonedds/pull/106/commits/756fa27bc547a00ad36b80327ef825185a205d87.